### PR TITLE
Add family_scheme__full_time_earner to minimum family tax credit in f…

### PR DESCRIPTION
**Pull request by verbman**
* Tax and benefit system evolution. | Technical improvement. 
* Impacted periods: all.
* Impacted areas: ` openfisca_aotearoa/variables/acts/income_tax/family_scheme/`
* Details:
  - Renamed `family_scheme__in_work_tax_credit_is_full_time_earner` to `family_scheme__full_time_earner`
  - Applied `family_scheme__full_time_earner` to `family_scheme__qualifies_for_minimum_family_tax_credit`

- - - -

These change:

- Impact the OpenFisca-Aotearoa public API (for instance renaming or removing a variable)
- Add a new feature (for instance adding a variable)
- Fix or improve an already existing calculation.
- Change non-functional parts of this repository (for instance editing the README)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+# 8.0.0 - [#125](https://github.com/ServiceInnovationLab/openfisca-aotearoa/pull/125)
```
Comment (Br3nda) [inline CHANGELOG.md]: I'm not sure we need a full version leap for this change.

File: openfisca_aotearoa/tests/income_tax/family_scheme/in_work_tax_credit.yaml [hunk 2]
```diff
@@ -68,7 +68,7 @@
       family_scheme__assessable_income:
         2018: 20000
       family_scheme__in_work_tax_credit_income_under_threshold: false
-      family_scheme__in_work_tax_credit_is_full_time_earner: true
+      family_scheme__full_time_earner: true
```
Comment (Br3nda) [inline openfisca_aotearoa/tests/income_tax/family_scheme/in_work_tax_credit.yaml]: Much better

File: openfisca_aotearoa/tests/income_tax/family_scheme/minimum_family_tax_credit.yaml [hunk 3]
```diff
@@ -41,5 +66,28 @@
   titled_properties:
     - id: "Whare"
       owners: ["Rakinui", "Tāne"]
+  output_variables:
+    family_scheme__qualifies_for_minimum_family_tax_credit: [False, False]
+- name: A Person would be qualified for the working for the minumum family tax credit but is not a full time earner
+  period: 2018-08
+  absolute_error_margin: 0
+  persons:
+    - id: "Rakinui"
```
Comment (Br3nda) [inline openfisca_aotearoa/tests/income_tax/family_scheme/minimum_family_tax_credit.yaml]: Wai pounamu :)

File: CHANGELOG.md (hunk 1 above)
Comment (verbman) [reply CHANGELOG.md]: Contributing guide says:
Major bump:    Renaming or removing a variable from the tax and benefit system.

File: openfisca_aotearoa/tests/income_tax/family_scheme/minimum_family_tax_credit.yaml (hunk 3 above)
Comment (verbman) [reply openfisca_aotearoa/tests/income_tax/family_scheme/minimum_family_tax_credit.yaml]: Āe

**Comment by Br3nda** [comment]
This pull request **introduces 1 alert** when merging 506979164509ac939e0f695803b4ca42744728c1 into bf22c805fbd55d0555e1d0f890c52b07464d06d3 - [view on LGTM.com](https://lgtm.com/projects/g/ServiceInnovationLab/openfisca-aotearoa/rev/pr-18855f47f61c62b16fb342dde727ef6e06e40bfd)

**new alerts:**

* 1 for Non\-callable called

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**